azure-pipelines.yml

```diff
@@ -77,17 +77,21 @@ stages:
       inputs:
         versionSpec: $(node_version)
       displayName: 'Install Node.js'
-    - script: yarn global add lerna
-      displayName: 'Install lerna'
     - script: |
-        yarn bootstrap
-        set CI=false
-        yarn build
-      name: Build
-    - script: |
-        set CI=true
-        yarn test
-      name: Test
+        powershell -Command "$ProgressPreference = 'SilentlyContinue'; iwr -outf %TEMP%\LLVM-8.0.0-win64.exe https://rust-lang-ci2.s3.amazonaws.com/rust-ci-mirror/LLVM-8.0.0-win64.exe"
```

Comment:
I am wondering if it makes sense to add this new script above the original script of `install lerna`, and keep the rest of the scripts as it is without removing them adding the `pwsh` script.

Reply:
I'll restore them if this script works.